geotrek/api/v2/views/tourism.py

```diff
@@ -99,3 +100,20 @@ def get_queryset(self):
                               ) \
             .annotate(geom_transformed=Transform(F('geom'), settings.API_SRID)) \
             .order_by('name')  # Required for reliable pagination
+
+
+class TouristicEventPlaceViewSet(api_viewsets.GeotrekGeometricViewset):
+    filter_backends = api_viewsets.GeotrekGeometricViewset.filter_backends + (
+        api_filters.NearbyContentFilter,
+        api_filters.UpdateOrCreateDateFilter,
+    )
+    serializer_class = api_serializers.TouristicEventPlaceSerializer
+    filteredset_view = ["pk", "geom", "name"]
```

Comment:
Fields should only be specified in `TouristicEventPlaceSerializer` in `Meta` class